### PR TITLE
Adding test for jax's _src frontend + fixing lint issues

### PR DESCRIPTION
**Pull request by Fayad-Alman**
Your task is to write tests for the functions of this frontend module https://github.com/unifyai/ivy/tree/master/ivy/functional/frontends/jax/_src.
It would be ideal to use hypothesis as much as possible, but in complex cases you will probably have to resort to plain assertions.

### CODE REVIEW

**Review by abdulasiraj — COMMENTED** [review]
Hi @Fayad-Alman,

tree_leaves function return the leaves of the tree. you have to come up with the solution that checks the resulting leaves with some ground truth.

Thanks!

**Comment by Fayad-Alman** [comment]
Thank you for the help @abdulasiraj 
I've been trying to fix the issue, but now I'm getting an error stating the tree_leaves function doesn't exist. Mind taking a look?
Thanks again!

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_src_tree_util.py [hunk 1]
```diff
@@ -43,6 +43,29 @@ def tree_dict_strategy(draw):
...
+def test_jax_tree_leaves(
+    *,
+    tree,
+    test_flags,
+    fn_tree,
+    frontend,
+    on_device,
+):
+    # Apply the tree_leaves function to obtain the leaves of the tree
+    result = ivy.Container(tree_leaves(tree, is_leaf=None))
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_src_tree_util.py:60]: You need to use the full path of the function or import it from our jax frontend. That's why you are getting the error.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by AnnaTz** [comment]
You seem to have done some kind of formatting to a bunch of files. Ideally the PR should affect the single test file you are working on, nothing else.

**Comment by Fayad-Alman** [comment]
@AnnaTz in that case I'll undo the commits related to fixing the lint tests and put those on a separate PR soon

**Pull request closed without merging**